### PR TITLE
More CSS changes

### PR DESCRIPTION
**Pull request by colin353**
- Inventory view looks a lot better. I have pre-rendered cards in there too
 - Fixed some CSS issues introduced with the refactoring.
 - When you click a card, you get a popover. There isn't an interface on there, but eventually
   you will be able to see the cost and activate the card from there.
 - Lots of holes in this PR, but seeing as it is an improvement, I think it may be OK
   to merge anyway.

### CODE REVIEW

File: js/View.elm [hunk 1]
```diff
@@ -27,8 +28,9 @@ view model =
 
 welcomeView : WelcomeModel -> Html WelcomeMsg
 welcomeView model =
-    div []
-        [ div [ class "box" ]
+    div [ class "welcome" ]
+        [ div [] []
```
Comment (iteloo) [inline js/View.elm]: Is this used?

File: js/View.elm [hunk 2]
```diff
@@ -68,21 +70,36 @@ gameView model =
...
-                        [ if model.inventoryVisible then
-                            [ inventoryView model.inventory ]
-                          else
-                            []
+                        [ [ inventoryView model.inventory ]
                         , [ toolbar model
-                          , div []
-                                [ text ("$" ++ (toString model.gold))
-                                ]
+                          , div [] []
```
Comment (iteloo) [inline js/View.elm]: What about this?

**Comment by iteloo** [comment]
I made some large changes. I'll wait for you to review them first.

**Comment by colin353** [comment]
Sorry for the super delay in responding here. I've been on vacation + sick. I'm mostly back to life now though, so I'll review this and do some more work this weekend.

**Comment by colin353** [comment]
Looks amazing. I've never heard of a zoomlist data structure before, haha, but it makes sense. I'm going to pull this in.

**Comment by iteloo** [comment]
It's normally called a zipper, and you can make one for any inductively defined data structure (for example, list and trees). I just named it zoom list to keep with your naming convention haha :P

**Comment by iteloo** [comment]
It think we should leave the PR number that Github inserts automatically in the commit message when we merge.

**Comment by colin353** [comment]
Okay, I'll leave it in in the future.

**Pull request merged**